src/custom/hooks/useApproveCallback.ts

```diff
@@ -4,7 +4,7 @@ import { Field } from '@src/state/swap/actions'
 import { computeSlippageAdjustedAmounts } from '@src/utils/prices'
 import { Trade } from '@uniswap/sdk'
 import { useMemo } from 'react'
-import { GP_ALLOWANCE_MANAGER_CONTRACT_ADDRESS } from '../constants'
+import { GP_ALLOWANCE_MANAGER_CONTRACT_ADDRESS } from 'constants/index'
```

Comment:
does this need `index` here?

Reply:
yep, it does. It returns sth else if not :)

Reply:
What about `constants/`?

Reply:
that works too, is that much of a difference? 🤷‍♂️

Reply:
Not at all